.github/workflows/release.yml

```diff
@@ -50,18 +50,19 @@ jobs:
           # Ensure we're on a clean main branch
           git checkout main
           git pull origin main
-          
+
           # Extract the new version of c2pa from the release PR output JSON
           NEW_VERSION=$(echo "$PR" | jq -r '.releases[] | select(.package_name == "c2pa") | .version')
-          
-          # Add a comment to the lib.rs file to trigger a change
-          echo "// Updated to use c2pa crate version $NEW_VERSION" >> c2pa_c_ffi/src/lib.rs
-          
+
+          # Update the c2pa dependency version in c2pa_c_ffi/Cargo.toml
+          sed -i.bak "s/\(c2pa = { path = \"..\/sdk\", version = \"\)[^\"]*\"/\1$NEW_VERSION\"/" c2pa_c_ffi/Cargo.toml
```

Comment:
I'm not sure if you can specify a path and version simultaneously, only the path should be needed here.

Reply:
Ah ok interesting, I didn't know that was possible.

Reply:
I think it relates to the fact we're in a workspace. There is a local path, but cargo on publish ignores local paths. (But  in local dev it uses your path and verifies the version still, I recall having had errors about mismatches when I poked at this).